### ISSUE TITLE
Incorrect History Sync Time

### ISSUE THREAD

**Issue by george-emerald** [tegon/traktflix#126]
I don't know if this is as configured, but the time in the History Sync section is always when I started the episode instead of when I finished it. Does it correct it when it sends it or does it say that I finished an episode when I started it?

**Comment by rafaelgssa** [comment]
Are you sure this isn't a Netflix problem?

**Comment by george-emerald** [comment]
@rafaelgssa I can't check that since you can only see the date in Netflix viewing history.

**Comment by Sparoney** [comment]
I have this issue too. I've noticed if you're mid way through watching something on Netflix then stop watching it and resume it later, it changes the time to when you started watching it again.

**Comment by MrMamen** [comment]
Yes. Netflix overwrites the dates at the last time out started playing any given episode or movie.

**Comment by smoothlystable** [comment]
The Netflix API returns "bookmark", how long you watched the episode/movie. So I guess one solution would be to add that time to Netflix date when syncing to Trakt. That would give the correct date and time for when you finished watching something, instead of start time.
